src/components/common/MessageBox/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/styles";
+
+import { useStyles } from "./styles";
+
+const backgroundColor = {
+  error: "error",
+  success: "success",
+};
+
+const MessageBox = ({ classes, errorMsg, type }) => {
```

Comment:
This could also be named as `Alert`.

Reply:
[done](https://github.com/Vivek205/refactored-dapp/pull/22/commits/ec99abdc8b56f0ccb4e224c4b635cf8633ac6716)